reference-implementation/to-upstream-wpts/writable-streams/start.js

```diff
@@ -15,17 +15,11 @@ async_test(t => {
       });
     },
     write(chunk) {
-      t.step(() => {
-        assert_true(expectWriteCall, 'write should not be called until start promise resolves');
-        assert_equals(chunk, 'a', 'chunk should be the value passed to write');
-        t.done();
-      });
+      assert_true(expectWriteCall, 'write should not be called until start promise resolves');
+      assert_equals(chunk, 'a', 'chunk should be the value passed to write');
```

Comment:
Hmm, I think we should keep this test async_test. assert_.* methods should be captured by t.step() or they'll be passed to the rejection callback for these underlying sink methods. I see that writer.ready would pass the thrown AssertionError to promise_test() finally, but it's depending on the internals of the WritableStream.